mmocr/apis/train.py

```diff
@@ -30,30 +30,31 @@ def train_detector(model,
     # prepare data loaders
     dataset = dataset if isinstance(dataset, (list, tuple)) else [dataset]
     # step 1: give default values and override (if exist) from cfg.data
-    loader_cfg = {
-        **dict(
-            seed=cfg.get('seed'),
-            drop_last=False,
-            dist=distributed,
-            num_gpus=len(cfg.gpu_ids)),
+    default_loader_cfg = dict(
+        num_gpus=len(cfg.gpu_ids),
+        dist=distributed,
+        seed=cfg.get('seed'),
+        drop_last=False,
+        persistent_workers=False)
+    # update overall dataloader(for train, val and test) setting
+    default_loader_cfg.update({
+        k: v
+        for k, v in cfg.data.items() if k not in [
+            'train', 'val', 'test', 'train_dataloader', 'val_dataloader',
+            'test_dataloader'
+        ]
+    })
+    train_loader_cfg = {
+        **default_loader_cfg,
         **({} if torch.__version__ != 'parrots' else dict(
                prefetch_num=2,
                pin_memory=False,
```

Comment:
This configuration was previously available in the val_dataloader, but now it is only available in the train_dataloader. Should we move them to default_lodaer_cfg?